data/harohar13.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Harohar",
+  "bio": "Software engineer,writer",
+  "links": [
+    {
+      "name": "Follow me on github",
+      "url": "https://github.com/harohar13",
+      "icon": "github"
```

Comment:
@harohar13 Consider specifying the icon set you would like displayed. We support the following icon sets: **Fa** and **Si** from React Icons. Take a look at our documentation on this topic [here](https://www.biodrop.io/docs/how-to-guides/socials-shortcuts-json) , specifically the section under "Add Social shortcuts"

Additionally in future perhaps consider adding more i.e links, testimonials, and milestones to your profile.